### PR TITLE
move annotation endpoint in API builder training

### PR DESCRIPTION
**Pull request by fharper**

### CODE REVIEW

**Comment by fharper** [comment]
I need to update the JSON examples in [train-using-the-annotation-endpoint.md](https://github.com/mindee/documentation/pull/321/files#diff-010b3b129874626d0e467c5b411b24bcf5f0c23efe1e594c0e81c2729fa03390), but that can give you an idea of the modifications I'll do based on our discussion @olivmindee

File: build-your-first-document-parsing-api/training-interface/train-using-the-annotation-endpoint.md [hunk 1]
```diff
@@ -2,89 +2,328 @@
...
-{
-  "codes": [
-    {
-      "code": "curl -X POST 'https://api.mindee.net/v1/products/doug1/us_w9/v1/predict?annotations=true'    -H 'Authorization: Token f7d710e56f215512f9fb4de536d5af8a'     -H 'content-type: multipart/form-data; boundary=----WebKitFormBoundary7MA4YWxkTrZu0gW' -F document=@harry_potter.pdf   ",
-      "language": "shell"
-    }
-  ]
-}
-[/block]
+When using the API, you can correct and upload the results back into the model to continue training. However, not every prediction is 100% accurate, so you may need to provide corrections. To get additional potential results, add the **training=true&with_candidate=true** URL parameters to the prediction API:
```
Comment (olivmindee) [inline build-your-first-document-parsing-api/training-interface/train-using-the-annotation-endpoint.md:6]: parameter `with_candidates` (with an s)

phrasing may also remind the user that he can still train using the training interface of the platform

File: build-your-first-document-parsing-api/training-interface/train-using-the-annotation-endpoint.md [hunk 2]
```diff
@@ -2,89 +2,328 @@
...
+
+```shell
+curl -X POST 'https://api.mindee.net/v1/products/_<account_name>_/_<api_name>_/_<api_version>_/predict?training=true&with_candidate=true' -H 'Authorization: Token my-api-key-here' -F /path/to/your/file.png
```
Comment (olivmindee) [inline build-your-first-document-parsing-api/training-interface/train-using-the-annotation-endpoint.md:9]: jump line for each arg

parameter `with_candidates` (with an s)

File: build-your-first-document-parsing-api/training-interface/train-using-the-annotation-endpoint.md [hunk 3]
```diff
@@ -2,89 +2,328 @@
...
+```
+
 This tells the API to provide alternative candidates for each prediction. Each prediction and candidate will now have a **candidate_key** that can be returned for training.
-[block:code]
+
```
Comment (olivmindee) [inline build-your-first-document-parsing-api/training-interface/train-using-the-annotation-endpoint.md:13]: The sample code provided is not explaining how to retrieve candidates but predictions. Candidates are to be found in `inference.pages[].extras.candidates` and describe the `extraction` and `classification` possible choices.

Create another block mentioning that predictions may be available if the API was previously trained (at the time the document was uploaded for training). A sample code with prediction is not useful.

File: build-your-first-document-parsing-api/training-interface/train-using-the-annotation-endpoint.md [hunk 4]
```diff
@@ -2,89 +2,328 @@
...
     }
-  ]
 }
-[/block]
-In the example above, the name "Harry" is correct, so we can return the *candidate_key* back with the label "name".  However the city value of ", CA" is incorrect, and we must find the correct words from the extracted set. 
+```
+
+In the example above, the name "Harry" is correct, so we can return the *candidate_key* back with the label "name".  However the city value of ", CA" is incorrect, and we must find the correct words from the extracted set.
 
 The JSON response has  set of assets “OCR ->Candidates” available for each page and each element that is detected. In this case of the city, we are looking for:
```
Comment (olivmindee) [inline build-your-first-document-parsing-api/training-interface/train-using-the-annotation-endpoint.md:249]: candidates are to be found in `inference.pages[].extras.candidates` and then used to create a label.

Need explanation of `extraction` & `classification` features. To be merged with annotations endpoint page content.

File: build-your-first-document-parsing-api/training-interface/train-using-the-annotation-endpoint.md [hunk 5]
```diff
@@ -2,89 +2,328 @@
...
 We can use these keys in the annotation response to Mindee.
-[block:api-header]
-{
-  "title": "Annotation endpoint: POST"
-}
-[/block]
+
+## Annotation endpoint: POST
+
 Now we can build the annotation update to Mindee for future training to improve the model. The documentId in the URL is provided in the JSON response, so the API knows which file the training is being improved for. Each label is named, and the candidate keys are submitted in the "selected" array:
```
Comment (olivmindee) [inline build-your-first-document-parsing-api/training-interface/train-using-the-annotation-endpoint.md:276]: Review phrasing + no classification field example

**Review by olivmindee — COMMENTED** [review]
Content must be rewritten to understand what is the candidates object for extraction & classification fields. It's worth mentioning that prediction can be available if the API was trained before the document was uploaded

**Comment by fharper** [comment]
Oh, you review everything. Thanks, that will be helpful! I wasn't clear, but I just wanted to be sure I moved the necessary part where it made sense, and have the right parameters, so I can updates the examples and text according, hence not everything was modified. I tried to save some time by having a quick check... failed 😬

It's fine, it will be useful, thanks again!

I need to redo all examples for the Node.js SDK invoice page with a new fake invoice with all extracted fields and I'm doing that right after, so ETA, tomorrow EOD it will finally be in production 🎉

**Pull request merged**